python/xorbits/core/data.py

```diff
@@ -18,6 +18,7 @@
 from typing import TYPE_CHECKING, Any, Callable, Dict, Iterable, List, Type
 
 from pandas.api.types import is_integer_dtype
+from pandas.core.dtypes.common import is_float_dtype
```

Comment:
The `is_float_dtype` can be imported from `pandas.api.types` as well. Normally, the `pandas.core` should not be imported which mentioned in pandas doc that, the core is not for users, api is.